app.go

```diff
@@ -60,7 +61,8 @@ func Loop(args []string, onReady func(*App)) error {
 	defer cerr.free()
 
 	app := App{
-		ready: make(chan struct{}),
+		ready:  make(chan struct{}),
+		server: newServer(http.DefaultServeMux),
```

Comment:
Hmm won't using `DefaultServeMux` here make it impossible to use third party http handlers, like `gorilla.Mux` etc? My suggestion would be:
- export `newServer` as `gallium.NewServer`
- require users to instantiate their own server if they want one
- require that users pass in their local server url explicitly as in `OpenWindow(myserver.BaseURL(), opts)`

That will also mean that users will have the flexibility to avoid opening any network ports at all if they don't want a server, or to have multiple servers if they want that (e.g. one per window). What do you think?